.github/workflows/platform-release.yaml

```diff
@@ -1,6 +1,6 @@
 name: Release with Platform builds
 
-on:
+on: 
```

Comment:
A few trailing spaces on lines in this file (in VS Code, use `cmd+k, cmd+z` to trim)